app/controllers/admin/content/system-images/list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  actions: {
+    openModal(selectedSubTopic) {
+      this.set('selectedSubTopic', selectedSubTopic);
+      this.set('isModalOpen', true);
```

Comment:
@niranjan94 I am bit confused then how would we achieve this.
```
{{#if selectedSubTopic}}
{{modals/change-image-modal subTopic=selectedSubTopic}}
{{/if}}
```
Like this? Please guide.